operations/mimir-mixin/config.libsonnet

```diff
@@ -89,6 +89,9 @@
       write: '(distributor|ingester.*|mimir-write.*)',
       read: '(query-frontend.*|querier.*|ruler-query-frontend.*|ruler-querier.*|mimir-read.*)',
       backend: '(ruler|query-scheduler.*|ruler-query-scheduler.*|store-gateway.*|compactor.*|alertmanager|overrides-exporter|mimir-backend.*)',
+
+      // Specific groups of components.
+      memory_map_areas: '(cortex|ingester.*|store-gateway.*|mimir-write.*|mimir_backend.*)',
```

Comment:
Not big fan of this cause adds more additional complexity to the `job_names`. I think you just want to merge `ingester` and `store_gateway`, so you could do the following in the alert (yes, there are some repetitions but it's not a bit deal):

```
$.jobMatcher('(%s|%s)' % [$._config.job_names.ingester, $._config.job_names.store_gateway])
```

Reply:
Fair enough, will do it that wayl